src/main/java/net/datafaker/service/RandomService.java

```diff
@@ -116,6 +116,11 @@ public byte[] nextRandomBytes(int numberOfBytes) {
         return randomBytes;
     }
 
+    public <T extends Enum<T>> T nextEnum(Class<T> klass) {
```

Comment:
Since this is now a public method, would it be possible to add a unit test for this?

Reply:
@bodiam Thanks, added the tests.